app/src/components/inputs/inputRadio/inputRadio.jsx

```diff
@@ -92,9 +96,6 @@ InputRadio.defaultProps = {
   onChange: () => {},
   onFocus: () => {},
   onBlur: () => {},
-  inputClassNames: {
-    togglerClassName: '',
-    childrenClassName: '',
-  },
   title: '',
+  mode: '',
```

Comment:
May be specify a meaningful default value, f.e. `mode: default`?

Reply:
If I get you right, we should wrap old styles with `mode-default` class?

Reply:
'default' mode value may be used in prop definitions just to specify the way how the mode should look like, without any special styles. But separation some specific, not common styles with `mode-default` sounds good too.